### DISCUSSION TITLE
[Kavita] Reading Lists could include parts of an issue to accomodate collected editions

### DISCUSSION THREAD

**Discussion by jncmney** [Kareadita/Kavita#3109] [Ideas]
### Idea Description

At the moment, Reading Lists are based on Issues. However, sometimes short stories or various issues are collected in a single publication and when a story line cuts across, it's not possible to create a Reading List in the right order.

For example, I have a storyline for which I have many individual Issues, but the short stories are collected in Annuals. Chronologically, these short stories go in between the main series.

I can't create a Reading List that includes the short stories in order because they're all in a single issue (cbz).

I would like to be able to create a Reading List that's something like this:

1. Issues 1-5 (all individual cbz files)
2. Short Story No. 1 (collected inside an "Annual 1" issue that's a single cbz and contains 4 short stories)
3. Issues 6-10 (all individual cbz files)
4. Short Story No. 2 (collected in the same "Annual 1" cbz as the short story above)
5. Issues 11-12

For this to work, we'd need to be able to define pages within a cbz for the Reading List, so the above would look like this:

1. Issue 1
2. Issue 2
3. Issue 3
4. Issue 4
6. Issue 5
7. Annual 1 - Pages 1-22
8. Issue 6
9. Issue 7
10. Issue 8
11. Issue 9
12. Issue 10
13. Annual 1 - Pages 23-45 
14. Issue 11
15. Issue 12

This would also be useful for those of us that collect TPBs, Compendiums, Deluxe Sets, and other collections.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This isn't feasible and seems like a rare edge case for how you're collecting files. I'll leave it up to see if this is something the CBL group is interested in.

**Reply by jncmney** [reply]
I'm not sure how often this happens, but here's a real example: I'm building a Reading List to include all of Star Wars Canon comics following this: https://youtini.com/timelines/canon/canon-comics#The-High-Republic

It starts with The High Republic Adventures : Quest of the Jedi, then all of Blade, then a few issues of The High Republic Adventures (2022), then a few of The High Republic (2022), then back to some issues of Adventures, etc... and then... **two short stories that are included in Star Wars – The High Republic Adventures Annual 2021**, then back to other regular issues from multiple series... and after a while, **we're back to other short stories in the Annual 2021**.

I think I'm a bit of a completionist.
